wheelhouse.txt

```diff
@@ -17,12 +19,12 @@ MarkupSafe<2.0.0;python_version < '3.6'
 MarkupSafe<2.1.0;python_version == '3.6' # Just for python 3.6
 MarkupSafe;python_version >= '3.7' # newer pythons
 
-setuptools<42;python_version < '3.8'
+setuptools<42;python_version <= '3.8'
 # https://github.com/juju-solutions/layer-basic/issues/201
-setuptools<62.2.0;python_version >= '3.8'
-setuptools-scm<=1.17.0;python_version < '3.8'
+setuptools<62.2.0;python_version > '3.8'
+setuptools-scm<=1.17.0;python_version <= '3.8'
 # https://github.com/pypa/setuptools_scm/issues/722
-setuptools-scm<7;python_version >= '3.8'
+setuptools-scm<7;python_version > '3.8'
```

Comment:
This pins setuptools and setuptool-scm even on later versions, they should be unpinned for py310.

e.g.

```
setuptools<42;python_version <= '3.8'
setuptools<62.2.0;python_version == '3.8'
setuptools;python_version > '3.8'

setuptools-scm<=1.17.0; python_version <= '3.8'
setuptools-scm<7; python_version == '3.8'
setuptools-scm; python_version > '3.8'
```

This way we allow future development without incurring unnecessary technical debt by pinning installs on, say, py311.

Reply:
Okay, removing pinning to setuptools<62.2.0 as it looks like it's fixed upstream https://github.com/pypa/setuptools_scm/issues/713

Reply:
Same goes with setuptools-scm https://github.com/pypa/setuptools_scm/issues/722, removing pinning setuptools-scm<7.

Reply:
These two were pinned here though https://github.com/juju-solutions/layer-basic/pull/202 & https://github.com/juju-solutions/layer-basic/pull/204